### PR TITLE
Code Review of Q6

### PR DESCRIPTION
**Pull request by Harshwardhanwtagline**
Screenshot 1944-11-25 at 3 48 42 AM
Code's Output

### CODE REVIEW

File: Q6.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def count_substring(string, substring):
+
+    #Using list comprehension and slicing , i have count the occcurence of substring in string. and then i have append 1 in list,
```
Comment (priteshtagline) [inline Q6.py:3]: Remove extra space of from comment that comes after `slicing ,`

File: Q6.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    #after the i have sum all using sum function
+    count = sum([1 for _ in range(0,len(string)) if string[_:len(substring)+_] == substring])
```
Comment (priteshtagline) [inline Q6.py:5]: You are using `_` in the code. So please give it a proper name, because `_` can be used when you don't have to use that.

Comment (himanshu-suthar-tagline) [inline Q6.py]: You are using the `_` in your code. Generally, `_` can be used when you don't have to use that in the code. Please give a proper name to this.

File: Q6.py (hunk 1 above)
Comment (himanshu-suthar-tagline) [inline Q6.py]: You have extra space between `slicing and ,` the `slicing ,` in the comment. please remove that.

File: Q6.py (hunk 2 above)
Comment (Harshwardhanwtagline) [reply Q6.py]: Done sir i have corrected the mistake

File: Q6.py (hunk 1 above)
Comment (Harshwardhanwtagline) [reply Q6.py]: Done sir

**Pull request merged**